package.json

```diff
@@ -66,13 +63,12 @@
     "autoprefixer": "^9.4.5",
     "babel-jest": "^24.5.0",
     "codecov": "^3.1.0",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.11.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "es-check": "^5.0.0",
     "eslint": "^5.16.0",
     "eslint-config-edx": "^4.0.4",
     "font-awesome": "^4.7.0",
-    "gatsby": "^2.1.31",
     "history": "^4.7.2",
     "husky": "^0.14.3",
     "identity-obj-proxy": "^3.0.0",
```

Comment:
Same as above, frontend-build will get this installed. See dependencies of frontend build here if you're unsure: https://github.com/edx/frontend-build/blob/master/package.json